packages/react/src/components/Slider/Slider.js

```diff
@@ -664,7 +664,7 @@ export default class Slider extends PureComponent {
                   required={required}
                   min={min}
                   max={max}
-                  step={step}
+                  step={1}
```

Comment:
I caught this in the original issue:

> ...the step value should be required to be "1" anytime an input field is provided

Does this mean when the input field is not provided it can be anything?

```suggestion
                  step={hideTextInput ? step : 1}
```

Reply:
That's what I originally had, but noticed that it was always `1` no matter what since the step on the slider is calculated elsewhere

Reply:
If `step` is not configurable on the `input`, would it block float values or the special `any` value that can be given? Based on the [MDN entry for `step`](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/step), it sounds like having it always `1` would only allow integer values.

Reply:
Just pushed an update that uses `float` values, but I didn't see any difference between the two. Let me know if I'm still missing something

Reply:
Thanks, that example is helpful - it shows that the values are calculated correctly for the slider, and the input updates correctly as well. The problem is when the input gets focus and then blurred the validation runs, there is a `stepMismatch`.


https://user-images.githubusercontent.com/3360588/185998650-6d1a69cf-4f79-4f8a-a3ac-a7fdbc6cefd4.mov

In this case I think the user would need to specify `step=".1"`.